### DISCUSSION TITLE
Get the latest 100 days data from createtime

### DISCUSSION THREAD

**Discussion by k2nneth** [prisma/prisma#13258] [Q&A]
Would like to GET the data with the latest day from the current day.  

Any samples or documents I can be referred for Prisma service?

**Comment by austincrim** [comment]
Hey @k2nneth 👋 ,

You will want to do something like [this example from our docs](https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#custom-engine-file-locations), after calculating what the date was 100 days ago:
```ts
const result = await prisma.post.findMany({
  where: {
    date_created: {
      gte: new Date(
        '2022-02-01T14:21:00+0200'
      ) /* Includes time offset for UTC */,
    }
  }
})
```

**Comment by k2nneth** [comment]
Hi,

I do tried this but I need is dynamic, not static.

On 11 May 2022, at 03:03, Austin Crim ***@***.***> wrote:

﻿

Hey @k2nneth<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fk2nneth&data=05%7C01%7C%7C88b110eaab5640b4b98b08da32c020b1%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637878098012361033%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ZWvncb6EF4WnsS1eyNU3WzUFuI4ZEV%2FBzIjTFpsmzxs%3D&reserved=0> 👋 ,

You will want to do something like this example from our docs<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.prisma.io%2Fdocs%2Freference%2Fapi-reference%2Fenvironment-variables-reference%23custom-engine-file-locations&data=05%7C01%7C%7C88b110eaab5640b4b98b08da32c020b1%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637878098012361033%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=p4KM2E%2F1osgACMbp1aCnEbPUGx%2F9QFV4MZlBKcQnGKQ%3D&reserved=0>, after calculating what the date was 100 days ago:

const result = await prisma.post.findMany({
  where: {
    date_created: {
      gte: new Date(
        '2022-02-01T14:21:00+0200'
      ) /* Includes time offset for UTC */,
    }
  }
})

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
